packages/api/db/sql/20240207214919_animal_union_batch_id_view.sql

```diff
@@ -0,0 +1,25 @@
+CREATE VIEW animal_union_batch_id_view AS
```

Comment:
Probably needs license as well

Reply:
ChatGPT said "Adding the GNU General Public License (GPL) or any other software license directly to SQL files isn't a common practice", so I didn't add it. (but I didn't do any further research)
Do you think we can just add it?

Reply:
I'll defer to @antsgar to decide for us! 😄